requirements/base.in

```diff
@@ -14,3 +14,5 @@ pypdf                   # PDF manipulation library
 reportlab               # PDF generation library
 openedx-completion-aggregator # Completion aggregation service
 edx_ace                 # Messaging library
+# TODO: Use the PyPI version of this package once it is available. Ensure to pin the minimum version.
+git+https://github.com/open-craft/learning-paths-plugin.git@agrendalath/bb-9241-refactor-keys
```

Comment:
TODO: Replace this once https://github.com/open-craft/learning-paths-plugin/pull/13 is merged.